### PR TITLE
style: use file format in tab (newly supported)

### PR DESCRIPTION
**Pull request by henryiii**
Follow up to #818, see https://github.com/pypa/packaging.python.org/pull/818#issuecomment-817303874.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 sphinx==3.2.0
 sphinx-autobuild==0.7.1
-sphinx-inline-tabs==2021.3.19b5
+sphinx-inline-tabs==2021.4.11b9
```
Comment (webknjaz) [inline requirements.txt:3]: Maybe try triple equals?
```suggestion
sphinx-inline-tabs === 2021.4.11b9
```

Comment (henryiii) [reply requirements.txt:3]: The problem is that sphinx-inline-tabs requires Python 3.8+. Can that be changed on the readthedocs config in the GUI? I don't see a `.readthedocs.yml` to change it in locally.

Comment (henryiii) [reply requirements.txt:3]: Actually, it looks like it can only be changed via a config file. The GUI choices are 2.x and 3.x.

**Comment by henryiii** [comment]
Added a configuration file to bump to 3.8 does fix it, though @pradyunsg, I should note that readthedocs still defaults to Python 3.7, so upgrading a sphinx extension to no longer support 3.7 cuts out quite a lot of "default" users.

**Review by webknjaz — APPROVED** [review]

**Pull request merged**